app/src/main/res/layout/activity_submit.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="?attr/activity_background"
-    android:orientation="vertical">
+                xmlns:app="http://schemas.android.com/apk/res-auto"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="?attr/activity_background"
+                android:orientation="vertical"
+                android:importantForAutofill="noExcludeDescendants">
```

Comment:
This will mark the view and all descendants as unimportant for autofill and prevent heuristics from trying to pry out the data for autofill.